api/pkg/sessionmanager/interface.go

```diff
@@ -85,33 +85,37 @@ const (
 )
 
 func (r RedisSessionManager) AddNotification(req *http.Request, w http.ResponseWriter, notification *Notification) error {
-	session, err := r.Store.Get(req, varNotifications)
-	if err != nil {
-		return err
-	}
-	session.AddFlash(notification)
-	if err := session.Save(req, w); err != nil {
-		return err
-	}
+	// FIXME was converted to noop because of securecookie error
```

Comment:
Have we created tickets for these FIXMEs in JIRA?

Reply:
Now we have :)